cortex/client.py

```diff
@@ -241,9 +170,10 @@ def client(api_endpoint: str = None, api_version: int = _DEFAULT_API_VERSION, ve
     @staticmethod
     def from_message(msg, verify_ssl_cert=None):
         """
-        Creates a Cortex client from a skill's input message, expects { api_endpoint:'..', token:'..', projectId:'..' }
```

Comment:
use standard json notation with double quotes 

OR

standard python dict notation with (double) quotes around keys and values